src/AirshipBindings.iOS.Core/ApiDefinitions.cs

```diff
@@ -0,0 +1,2852 @@
+        string Build ();
+    }
+
+    // @interface UAMediaEventTemplate : NSObject
+    [BaseType(typeof(NSObject))]
+    interface UAMediaEventTemplate
+    {
+        // @property (readwrite, copy, nonatomic, nullable) NSString *identifier;
+        [NullAllowed, Export("identifier")]
+        string Identifier { get; set; }
+
+        // @property (readwrite, copy, nonatomic, nullable) NSString *category;
+        [NullAllowed, Export("category")]
+        string Category { get; set; }
+
+        // @property (readwrite, copy, nonatomic, nullable) NSString *type;
+        [NullAllowed, Export("type")]
+        string Type { get; set; }
+
+        // @property (readwrite, copy, nonatomic, nullable) NSString *eventDescription;
+        [NullAllowed, Export("eventDescription")]
+        string EventDescription { get; set; }
+
+        // @property (assign, readwrite, nonatomic) BOOL isFeature;
+        [Export("isFeature")]
+        bool IsFeature { get; set; }
+
+        // @property (readwrite, copy, nonatomic, nullable) NSString *author;
+        [NullAllowed, Export("author")]
+        string Author { get; set; }
+
+        // @property (readwrite, copy, nonatomic, nullable) NSString *publishedDate;
+        [NullAllowed, Export("publishedDate")]
+        string PublishedDate { get; set; }
+
+        // + (nonnull instancetype)browsedTemplate;
+        [Static]
+        [Export("browsedTemplate")]
+        UAMediaEventTemplate BrowsedTemplate ();
+
+        // + (nonnull instancetype)starredTemplate;
+        [Static]
+        [Export("starredTemplate")]
+        UAMediaEventTemplate StarredTemplate ();
+
+        // + (nonnull instancetype)sharedTemplate;
+        [Static]
+        [Export("sharedTemplate")]
+        UAMediaEventTemplate SharedTemplate ();
+
+        // + (nonnull instancetype)sharedTemplateWithSource:(nullable NSString *)source withMedium:(nullable NSString *)medium;
+        [Static]
+        [Export("sharedTemplateWithSource:withMedium:")]
+        UAMediaEventTemplate SharedTemplate ([NullAllowed] string source, [NullAllowed] string medium);
+
+        // + (nonnull instancetype)consumedTemplate;
+        [Static]
+        [Export("consumedTemplate")]
+        UAMediaEventTemplate ConsumedTemplate ();
+
+        // + (nonnull instancetype)consumedTemplateWithValueFromString: (nullable NSString *)eventValue;
+        [Static]
+        [Export("consumedTemplateWithValueFromString:")]
+        UAMediaEventTemplate ConsumedTemplate ([NullAllowed] string eventValue);
+
+        // + (nonnull instancetype)consumedTemplateWithValue: (nullable NSNumber *)eventValue;
+        [Static]
+        [Export("consumedTemplateWithValue:")]
+        UAMediaEventTemplate ConsumedTemplate ([NullAllowed] NSNumber eventValue);
+
+        // - (nonnull UACustomEvent *)createEvent;
+        [Export("createEvent")]
+        UACustomEvent CreateEvent ();
+    }
+
+    // @interface UAModifyTagsAction : UAAction
+    [BaseType(typeof(UAAc
```

Comment:
Can you do a search/replace for "Deprecated ?" -> "Deprecated -"

There's a ticket to fix the issue that's currently making the question mark appear.

Reply:
Good catch. Forgot about that one.

Reply:
FYI - I checked the iOS SDK source, and it does indeed contain a unicode "-" which Xamboni converts into that "?". We should probably have a ticket to clean unicode out of the SDK.

Reply:
There's already a ticket for that.